model/patch_lifecycle.go

```diff
@@ -537,7 +537,7 @@ func AbortPatchesWithGithubPatchData(createdBefore time.Time, closed bool, newPa
 
 	catcher := grip.NewSimpleCatcher()
 	for i, _ := range patches {
-		if patches[i].Version != "" {
+		if patches[i].Version != "" && patches[i].Alias != evergreen.CommitQueueAlias {
```

Comment:
you could make the argument that if someone pushes to a pr that has a running cq merge, the active merge should abort. Thinking about this more I'm actually torn if we should either leave the merge as is, or dequeue it for the user

Reply:
I think it makes sense to dequeue it for the user; one day someone is probably going to be annoyed that it didn't